### PR TITLE
completed demo-tour feature

### PR DESCRIPTION
**Pull request by Durgesh00**
Ticket:-https://trello.com/c/1L5X1kbL/362-demo-walkthrough-on-first-login

### CODE REVIEW

File: src/modules/admin/sideNav/SideNavContainer/index.jsx [hunk 1]
```diff
@@ -94,19 +95,44 @@ const SideNavContainer = () => {
       navPath: `${ROUTES.ADMIN}${ADMIN_ROUTES.FEEDBACKS}`,
     },
   ];
-
+  const steps = [
```
Comment (somnath77) [inline src/modules/admin/sideNav/SideNavContainer/index.jsx:98]: Remove this if it is unnecessary

File: src/modules/admin/sideNav/SideNavContainer/index.jsx [hunk 2]
```diff
@@ -94,19 +95,44 @@ const SideNavContainer = () => {
...
-      showProfileModal={profileModal}
-      searchModal={searchModal}
-      handleClearState={handleClearState}
-      handleSelectedNavItem={handleSelectedNavItem}
-      enabledSideNav={enabledSideNav}
-      isHeighlightQuestion={isHeighlightQuestion}
-      navLinks={navLinks}
-    />
+    <>
+      {/* <Tour
```
Comment (somnath77) [inline src/modules/admin/sideNav/SideNavContainer/index.jsx:119]: remove commented code

File: src/modules/admin/sideNav/SideNavContainer/index.jsx (hunk 1 above)
Comment (Durgesh00) [reply src/modules/admin/sideNav/SideNavContainer/index.jsx:98]: done

File: src/modules/admin/sideNav/SideNavContainer/index.jsx [hunk 3]
```diff
@@ -96,17 +96,19 @@ const SideNavContainer = () => {
...
-      handleSearchClick={handleSearchClick}
-      showProfileModal={profileModal}
-      searchModal={searchModal}
-      handleClearState={handleClearState}
-      handleSelectedNavItem={handleSelectedNavItem}
-      enabledSideNav={enabledSideNav}
-      isHeighlightQuestion={isHeighlightQuestion}
-      navLinks={navLinks}
-    />
+    <>
```
Comment (somnath77) [inline src/modules/admin/sideNav/SideNavContainer/index.jsx:99]: unnecessary empty brackets can be removed

**Review by somnath77 — APPROVED** [review]

**Review by somnath77 — DISMISSED** [review]

**Review by somnath77 — APPROVED** [review]

**Pull request merged**